packages/framework/src/Resources/views/Admin/Content/TransportAndPayment/freeTransportAndPaymentLimitSetting.html.twig

```diff
@@ -35,12 +35,15 @@
                     {% set domainName = domain.getDomainConfigById(domainId).getName() %}
                     <tr id="{{ priceLimitForm.vars.id }}" class="js-free-transport-and-payment-price-limit" data-domain-id="{{ domainId }}">
                         {% if isMultidomain() %}
-                            <td>{{ domainName }}</td>
+                            <td>
+                                {{ domainName }}
+                            </td>
                         {% endif %}
                         <td>
                             {{ form_widget(priceLimitForm.enabled, { attr: { class: 'js-free-transport-and-payment-price-limit-enabled' } } ) }}
                         </td>
-                        <td class="form-inline">
+                        <td class="form-inline in-overlay">
```

Comment:
Why? Is this necessary? I have tested it and saw the behavior without the `in-overlay` class but I am not sure if this is a proper solution, maybe you could discuss it with @kulincz

Reply:
seems that without position relative it is outside of the box
![image](https://user-images.githubusercontent.com/39240194/40846063-cf8df450-65b8-11e8-9b86-00a9a7fd2707.png)